tools/c7n_mailer/c7n_mailer/azure/azure_queue_processor.py

```diff
@@ -79,6 +79,31 @@ def process_azure_queue_message(self, encoded_azure_queue_message):
             queue_message['policy']['name'],
             ', '.join(queue_message['action'].get('to'))))
 
+        if any(e.startswith('slack') or e.startswith('https')
```

Comment:
i'm a little uncomfortable with a naked https being identified as slack, vs explicit protocol, if thats whats been done in the other processors so be it :-(..

there also a degree of duplication here that suggest we need a cleaner refactoring of base queue processor functionality.

Reply:
Yeah I did wonder about this too - I'd be inclined to remove that if it doesn't have some importance I'm missing.  Common user error is to start every URL with `https` instead of `azure` or `asq` or `s3` or whatever perhaps this would catch unintended things?

Reply:
I believe we can update both processors to search for `https://hooks.slack.com`. However, I don't know if someone uses short urls to hide hooks.slack.com for whatever reason

Reply:
And I totally agree with duplication. I just didn't want to have both new functions & refactoring in the same PR.. I was thinking to move this to some queue_processor code & allow Azure, AWS to implement some reading from queue & configure supported delivery methods (this will benefit gcp and other providers in the future)

Reply:
Replaced https with https://hooks.slack.com -- slack_delivery already uses this.